src/ai/backend/manager/models/kernel.py

```diff
@@ -982,13 +1001,17 @@ async def resolve_commit_status(self, info: graphene.ResolveInfo) -> Optional[st
 
     async def resolve_abusing_reports(
         self, info: graphene.ResolveInfo
-    ) -> Iterable[Optional[Mapping[str, Any]]]:
+    ) -> Iterable[Mapping[str, Any]]:
         containers = self.containers
         if containers is None:
             containers = await self.resolve_containers(info)
         if containers is None:
             return []
-        return [(await con.resolve_abusing_report(info, self.access_key)) for con in containers]
+        return [
+            result
+            for con in containers
+            if (result := await con.resolve_abusing_report(info, self.access_key)) is not None
+        ]
```

Comment:
Could we rewrite this part using `aiotools.TaskGroup`?

Reply:
I use TaskGroup here. thanks!